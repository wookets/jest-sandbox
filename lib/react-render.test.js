
import React from 'react';
import Link from './Link-component';
import renderer from 'react-test-renderer';

test('Link renders correctly', () => {
	const tree = renderer.create(
		<Link page="http://www.facebook.com">Facebook</Link>
	).toJSON();
	expect(tree).toMatchSnapshot();
});
