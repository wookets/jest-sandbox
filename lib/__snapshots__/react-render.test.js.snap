exports[`test Link renders correctly 1`] = `
<a
  className="normal"
  href="http://www.facebook.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}>
  Facebook
</a>
`;
